nima/http2/webserver/src/main/java/io/helidon/nima/http2/webserver/Http2Config.java

```diff
@@ -66,6 +59,26 @@ public interface Http2Config {
     @ConfiguredOption("8192")
     long maxConcurrentStreams();
 
+    /**
+     * This setting indicates the sender's maximum window size in bytes for connection-level flow control.
+     * Default and maximum value is 2<sup>31</sup>-1 = 2147483647 bytes. This setting affects the window size
+     * of HTTP/2 connection.
+     * Any value greater than 2147483647 causes an error. Any value smaller than initial window size causes an error.
+     * See RFC 9113 section 6.9.1 for details.
+     *
+     * @return maximum window size in bytes
+     */
+    @ConfiguredOption("65635")
+    int initialWindowSize();
+
+    /**
+     * Outbound flow control blocking timeout in milliseconds.
+     *
+     * @return duration
+     */
+    @ConfiguredOption("100")
+    long flowControlTimeout();
```

Comment:
Timeout method without `Duration` should define units in its name (e.g. `flowControlTimeoutMillis`).
Method with `Duration` as a parameter must also exist on builder

Reply:
Changed to Duration, can be configured with ISO_8601 duration text representation, tested in io.helidon.nima.http2.webserver.ConnectionConfigTest#flowControlTimeout